env.sh

```diff
@@ -1,5 +1,6 @@
 if [ "$1" = "Timber" ]
 then
+    export _JAVA_OPTIONS="-Xss2m"
```

Comment:
the temporary fix for pytimber to work

fwd. the mail from Riccardo and Danilo ;D